assets/js/mlmodels.js

```diff
@@ -0,0 +1,98 @@
+const { TypesenseInstantSearchAdapter, instantsearch } = window;
+
+const model = document.getElementsByClassName("mr-model")[0].id;
+console.log(model)
+const typesenseInstantsearchAdapter = new TypesenseInstantSearchAdapter({
+  server: {
+    apiKey: "Qhooem9HCRuFMVZPNQOhABAdEWJaSnlY", // Be sure to use an API key that only allows search operations
+    nodes: [
+      {
+        host: "cgnvrk0xwyj9576lp-1.a1.typesense.net",
+        port: "443",
+        protocol: "https",
+      },
+    ],
+    cacheSearchResultsForSeconds: 2 * 60, // Cache search results from server. Defaults to 2 minutes. Set to 0 to disable caching.
+  },
+  // The following parameters are directly passed to Typesense's search API endpoint.
+  //  So you can pass any parameters supported by the search endpoint below.
+  //  query_by is required.
+  additionalSearchParameters: {
+    query_by: "model-id,description",
```

Comment:
You need to use the same field name as in the collection
```suggestion
    query_by: "model_id,description",
```